### ISSUE TITLE
加载模型后点击聊天不显示聊天界面

### ISSUE THREAD

**Issue by Lihwnlp** [xorbitsai/inference#2859]
如图，当我点击聊天后不显示界面。

![Image](https://github.com/user-attachments/assets/e519e8fa-566c-4ce8-a469-13916726b6b3)
![Image](https://github.com/user-attachments/assets/7898c7f1-c7be-4fd7-abd7-b7bb8220a24c)

以下是我的环境
accelerate                    1.3.0
aiofiles                      23.2.1
aiohappyeyeballs              2.4.6
aiohttp                       3.11.12
aioprometheus                 23.12.0
aiosignal                     1.3.2
airportsdata                  20241001
aliyun-python-sdk-core        2.16.0
aliyun-python-sdk-kms         2.16.5
annotated-types               0.7.0
antlr4-python3-runtime        4.9.3
anyio                         4.8.0
argcomplete                   3.5.3
async-timeout                 5.0.1
attrdict                      2.0.1
attrs                         25.1.0
audioread                     3.0.1
auto_gptq                     0.7.1
autoawq                       0.2.5
autoawq_kernels               0.0.9
av                            14.1.0
bcrypt                        4.2.1
beautifulsoup4                4.13.3
bitsandbytes                  0.45.2
black                         25.1.0
boto3                         1.28.64
botocore                      1.31.85
cachetools                    5.5.1
cbor                          1.0.0
certifi                       2025.1.31
cffi                          1.17.1
charset-normalizer            3.4.1
chattts                       0.2.2
click                         8.1.8
cloudpickle                   3.1.1
colorama                      0.4.6
coloredlogs                   15.0.1
conformer                     0.3.2
controlnet_aux                0.0.7
crcmod                        1.7
cryptography                  44.0.0
datamodel-code-generator      0.27.2
datasets                      3.2.0
decorator                     5.1.1
diffusers                     0.32.2
dill                          0.3.8
diskcache                     5.6.3
distro                        1.9.0
ecdsa                         0.19.0
editdistance                  0.8.1
einops                        0.8.1
einx                          0.3.0
encodec                       0.1.1
eva-decord                    0.6.1
fastapi                       0.110.3
ffmpy                         0.5.0
filelock                      3.17.0
FlagEmbedding                 1.3.4
flatbuffers                   25.2.10
frozendict                    2.4.6
frozenlist                    1.5.0
fsspec                        2024.9.0
funasr                        1.1.16
fvcore                        0.1.5.post20221221
gdown                         5.2.0
gekko                         1.2.1
genson                        1.3.0
gguf                          0.14.0
gradio                        5.1.0
gradio_client                 1.4.0
h11                           0.14.0
hiredis                       3.1.0
httpcore                      1.0.7
httpx                         0.28.1
huggingface-hub               0.28.1
humanfriendly                 10.0
hydra-core                    1.3.2
HyperPyYAML                   1.2.2
idna                          3.10
ijson                         3.3.0
imageio                       2.37.0
imageio-ffmpeg                0.6.0
importlib_metadata            8.6.1
inflect                       5.6.2
inscriptis                    2.5.3
interegular                   0.3.3
iopath                        0.1.10
ir_datasets                   0.5.9
isort                         6.0.0
jaconv                        0.4.0
jamo                          0.4.1
jieba                         0.42.1
Jinja2                        3.1.5
jiter                         0.8.2
jj-pytorchvideo               0.1.5
jmespath                      0.10.0
joblib                        1.4.2
jsonschema                    4.23.0
jsonschema-specifications     2024.10.1
kaldiio                       2.18.0
lark                          1.2.2
lazy_loader                   0.4
libnacl                       2.1.0
librosa                       0.10.2.post1
lightning                     2.5.0.post0
lightning-utilities           0.12.0
llama_cpp_python              0.3.7
llvmlite                      0.44.0
loguru                        0.7.3
loralib                       0.1.2
lxml                          5.3.1
lz4                           4.4.3
markdown-it-py                3.0.0
MarkupSafe                    2.1.5
mdurl                         0.1.2
modelscope                    1.22.3
mpmath                        1.3.0
msgpack                       1.1.0
multidict                     6.1.0
multiprocess                  0.70.16
mypy-extensions               1.0.0
natsort                       8.4.0
nest-asyncio                  1.6.0
networkx                      3.4.2
numba                         0.61.0
numpy                         1.26.4
nvidia-ml-py                  12.570.86
omegaconf                     2.3.0
onnxruntime                   1.20.1
openai                        1.61.1
opencv-python                 4.11.0.86
optimum                       1.24.0
orjson                        3.10.15
ormsgpack                     1.7.0
oss2                          2.19.1
outlines                      0.1.14
outlines_core                 0.1.26
packaging                     24.2
pandas                        2.2.3
parameterized                 0.9.0
passlib                       1.7.4
pathspec                      0.12.1
peft                          0.14.0
pillow                        10.4.0
pip                           25.0
pipx                          1.7.1
platformdirs                  4.3.6
pooch                         1.8.2
portalocker                   3.1.1
propcache                     0.2.1
protobuf                      5.29.3
psutil                        6.1.1
pyarrow                       19.0.0
pyasn1                        0.6.1
pybase16384                   0.3.8
pycountry                     24.6.1
pycparser                     2.22
pycryptodome                  3.21.0
pydantic                      2.10.6
pydantic_core                 2.27.2
pydub                         0.25.1
Pygments                      2.19.1
pynndescent                   0.5.13
pypinyin                      0.53.0
pyreadline3                   3.5.4
PySocks                       1.7.1
python-dateutil               2.9.0.post0
python-jose                   3.3.0
python-multipart              0.0.20
pytorch-lightning             2.5.0.post0
pytorch-wpe                   0.0.1
pytz                          2025.1
pywin32                       308
PyYAML                        6.0.2
quantile-python               1.1
qwen-vl-utils                 0.0.10
redis                         5.2.1
referencing                   0.36.2
regex                         2024.11.6
requests                      2.32.3
rich                          13.9.4
rouge                         1.0.1
rpds-py                       0.22.3
rsa                           4.9
ruamel.yaml                   0.18.10
ruamel.yaml.clib              0.2.12
ruff                          0.9.6
s3transfer                    0.7.0
safehttpx                     0.1.6
safetensors                   0.5.2
scikit-image                  0.25.1
scikit-learn                  1.6.1
scipy                         1.15.1
semantic-version              2.10.0
sentence-transformers         3.4.1
sentencepiece                 0.2.0
setproctitle                  1.3.4
setuptools                    75.8.0
shellingham                   1.5.4
silero-vad                    5.1.2
six                           1.17.0
sniffio                       1.3.1
soundfile                     0.13.1
soupsieve                     2.6
soxr                          0.5.0.post1
sse-starlette                 2.1.3
starlette                     0.37.2
sympy                         1.13.1
tabulate                      0.9.0
tblib                         3.0.0
tensorboardX                  2.6.2.2
tensorizer                    2.9.1
termcolor                     2.5.0
threadpoolctl                 3.5.0
tifffile                      2025.1.10
tiktoken                      0.8.0
timm                          1.0.14
tokenizers                    0.21.0
tomli                         2.2.1
tomlkit                       0.12.0
torch                         2.5.1+cu124
torch-complex                 0.4.4
torchaudio                    2.5.1+cu124
torchdiffeq                   0.2.5
torchmetrics                  1.6.1
torchvision                   0.20.1+cu124
tqdm                          4.67.1
transformers                  4.48.3
transformers-stream-generator 0.0.5
trec-car-tools                2.6
typer                         0.15.1
typing_extensions             4.12.2
tzdata                        2025.1
umap-learn                    0.5.7
unlzw3                        0.2.3
urllib3                       2.0.7
userpath                      1.9.2
uvicorn                       0.34.0
vector-quantize-pytorch       1.17.3
verovio                       5.0.0
vocos                         0.1.0
warc3-wet                     0.2.5
warc3-wet-clueweb09           0.2.5
websockets                    12.0
wheel                         0.45.1
win32_setctime                1.2.0
x-transformers                2.0.2
xinference                    0.15.2
xoscar                        0.4.6
xxhash                        3.5.0
yacs                          0.1.8
yarl                          1.18.3
zipp                          3.21.0
zlib-state                    0.1.9
zstandard                     0.23.0
